.circleci/config.yml

```diff
@@ -141,18 +141,30 @@ workflows:
       - test:
           requires: [build]
           matrix:
+            alias: test-no-tl
             parameters:
-              suite: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13]
+              suite: [0, 1, 2, 3, 4, 5, 6, 7, 9, 10, 11, 12, 13]
           filters:
             tags:
               only: /.*/
+      - test:
+          requires: [build]
+          matrix:
+            alias: test-tl
+            parameters:
+              suite: [8]
```

Comment:
can we at least reshuffle the containers so this is at the end of the list?

Reply:
so I refrained from doing that before because the last container is special, it doesn't run a test. It has `compileJava` and `compileTestJava`. 

it is not included in the list here https://github.com/palantir/atlasdb/blob/f2f0426e8a070534b22383199b16d1f7bf845629/scripts/circle-ci/run-circle-tests.sh#L46

moving 8 to the back would make that awkward (I suppose I can move it to before the compile container but neutral on that)

let me know what you'd prefer

Reply:
yeah that's true. Maybe let's swap what 8 and 12 do (and above leave a comment that I guess now _12_ is intentionally excluded)?